### DISCUSSION TITLE
I want sortingFn to do nothing

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [TanStack/table#5197] [Q&A] [answered]
so I want when I click on the headers in columns it should show sort icon if true or false like acs or desc only for this I want  keep this code. but problem is I am sorting using api response I don't want local sorting but I still want to have the sorting acs, desc boolean for showing icon so need the sorting state. how can I do it

```ts
  const [sorting, setSorting] = React.useState<SortingState>([])
  const table = useReactTable({
    data,
    
    columns,
    state: {
      sorting
    },
    onSortingChange: setSorting,
```

**Comment by KevinVandy** [comment, marked as answer]
That's what the manualSorting option is exactly for

**Reply by 0xMALVEE** [reply]
thank you. you are life saver. 😘
